docs/platforms/javascript/guides/vue/features/component-tracking.mdx

```diff
@@ -18,87 +18,86 @@ To set up component tracking, you need to configure tracing. For details on how
 
-### Child Components
+### Child Component Tracking
 
 You can also track your app's child components to get more details about the rendering process. This feature will create spans for each tracked component instance. The spans are called **`ui.vue.[hook]`** where `[hook]` is replaced by each tracked lifecycle stage. For example, the span representing the mount stage (the time between `beforeMount` and `mounted`) is called `ui.vue.mount`.
 
-To set it up, add, at minimum, [`trackComponents`](#trackcomponents) in your `Sentry.init` call. Optionally, you can also add [`hooks`](#hooks), and [`timeout`](#timeout).
+To set it up, add the Vue Integration to your `Sentry.init()` call and, set the `tracingOptions.trackComponents` option.
+Set the `trackComponent` option to `true` to track all of your child components, or specify a list of individual comopnents you want to track:
 
-#### `trackComponents`
+```javascript {5-17}
+import * as Sentry from "@sentry/vue";
 
-This is the main option that controls which child components should be tracked. Set it to `true` to track all of them or specify a list of individual components you want to track:
-
-```javascript
 Sentry.init({
-  // ...
-  trackComponents: true,
-  // OR
-  trackComponents: [
-    "App",
-    "RwvHeader",
-    "RwvFooter",
-    "RwvArticleList",
-    "Pagination",
+  integrations: [
+    Sentry.vueIntegration({
+      tracingOptions: {
+        trackComponents: true,
+        // OR
+        trackComponents: [
+          "App",
+          "RwvHeader",
+          "RwvFooter",
+          "RwvArticleList",
+          "Pagination",
+        ],
+      },
+    }),
   ],
 });
 ```
 
-The default is `false`.
+The default value for `trackComponents` is `false`.
+
+#### Track Specific Component Lifecycle Hooks
 
-#### `hooks`
+You can control which lifecycle hooks should be tracked. This is helpful if, for example, you want to know if some components are removed during the initial page load, in which case you can configure the integration to also track `unmount` hooks:
 
-Control which lifecycle hooks should be tracked. This is helpful if, for example, you want to know if some components are removed during the initial page load, in which case you can add a `unmount` hook to the default:
+```javascript {8}
+import * as Sentry from "@sentry/vue";
 
-```javascript
 Sentry.init({
-  // ...
-  trackComponents: true
-  hooks: ["mount", "update", "unmount"],
+  integrations: [
+    Sentry.vueIntegration({
+      tracingOptions: {
+        trackComponents: true
+        hooks: ["mount", "update", "unmount"],
+      },
+    }),
+  ],
 });
 ```
 
-The following hooks are available to track in Vue 3: `['activate', 'create', 'unmount', 'mount', 'update']`
+The default set of tracked hooks is `['activate', 'mount', 'update']`.
```

Comment:
There's another item in the v9 list to remove `update` as a hook to track by default, right? No action required, was just remembering that we discussed this